test/jest/simple/make.webpack.config.js

```diff
@@ -0,0 +1,15 @@
+const path = require('path');
```

Comment:
Super nit but can we rename`test` dir to `tests` to match our repo's naming conventions?